.eslintrc

```diff
@@ -1,3 +1,11 @@
 {
-  "extends": "standard"
+  "extends": "standard",
+  "parserOptions": {
+    "sourceType": "script"
```

Comment:
The issue — standard config is not correct. See https://github.com/feross/eslint-config-standard/blob/master/eslintrc.json#L8 

@feross Could you please comment why it is so? 

https://github.com/eslint/eslint/issues/5301#issuecomment-184720161 is quite clear. We should not set `module` — we don't use ES6.